uploader.json

```diff
@@ -443,13 +488,7 @@
         },
         {
             "command": "/sbin/vgdisplay -vv",
-            "pattern": [
-                "Clustered",
-                "Couldn't find device with uuid",
-                "LV Name",
-                "Mirrored volumes",
-                "VG Name"
-            ]
+            "pattern": []
```

Comment:
This is probably a mistake where the filters didn't get applied properly.

Reply:
It's OK to remove these filters since the `vgdisplay` parse is updated and should not be filtered anymore.
See:
https://github.com/RedHatInsights/insights-core/pull/272
https://github.com/RedHatInsights/insights-plugins/pull/283

Reply:
Great! Makes sense to me.